src/Container.php

```diff
@@ -196,6 +196,6 @@ public function define_services(): void {
 		 *
 		 * @since 1.0.0
 		 */
-		do_action( 'rtcamp.google_login_services', $this );
+		do_action( 'rtcamp.google_login_services', $this ); // phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores -- Ignore as currently cannot change because of backward compatibility.
```

Comment:
What about disabling this rule in phpcs config?

```diff
<rule ref="WordPress-Core">
	<exclude name="Generic.Arrays.DisallowShortArraySyntax" />
	<exclude name="Generic.Commenting.DocComment.MissingShort" />
	<exclude name="WordPress.PHP.DisallowShortTernary" />
+	<exclude name="WordPress.NamingConventions.ValidHookName.UseUnderscores" />
</rule>
```

Probably include a todo comment as well?
```xml
<!-- @todo: Remove this once we rename the hooks with underscores and deprecate the old ones. -->
```

Reply:
Yes, We can do exclude it from there with a todo for future, works well. 👍